modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -53,45 +55,24 @@ endif::openshift-dedicated[]
 ifndef::openshift-rosa,openshift-dedicated[]
 * Azure: `LoadBalancerService` (with External scope)
 * Google Cloud Platform (GCP): `LoadBalancerService` (with External scope)
-* Bare metal: `NodePortService`
-* Other: `HostNetwork`
 endif::openshift-rosa,openshift-dedicated[]
-+
-[NOTE]
-====
-`HostNetwork` has a `hostNetwork` field with the following default values for the optional binding ports: `httpPort: 80`, `httpsPort: 443`, and `statsPort: 1936`.
-With the binding ports, you can deploy multiple Ingress Controllers on the same node for the `HostNetwork` strategy.
 
-.Example
-[source,yaml]
-----
-apiVersion: operator.openshift.io/v1
-kind: IngressController
-metadata:
-  name: internal
-  namespace: openshift-ingress-operator
-spec:
-  domain: example.com
-  endpointPublishingStrategy:
-    type: HostNetwork
-    hostNetwork:
-      httpPort: 80
-      httpsPort: 443
-      statsPort: 1936
-----
-====
-+
-[NOTE]
-====
-On {rh-openstack-first}, the `LoadBalancerService` endpoint publishing strategy is only supported if a cloud provider is configured to create health monitors. For {rh-openstack} 16.2, this strategy is only possible if you use the Amphora Octavia provider.
-
-For more information, see the "Setting cloud provider options" section of the {rh-openstack} installation documentation.
-====
 ifndef::openshift-rosa[]
 For most platforms, the `endpointPublishingStrategy` value can be updated. On GCP, you can configure the following `endpointPublishingStrategy` fields:
 
 * `loadBalancer.scope`
 * `loadbalancer.providerParameters.gcp.clientAccess`
+endif::openshift-rosa[]
+
+ifndef::openshift-rosa,openshift-dedicated[]
+For non-cloud environments, such as a bare-metal platform, use the `NodePortService` or `HostNetwork` fields` to configure the endpoint publishing strategy for your Ingress Controller.
+
+If you do not set a value in one of these fields, the default value is based on binding ports specified in the `infrastructure.config.openshift.io/cluster` `.status.platform` resource.
+endif::openshift-rosa,openshift-dedicated[]
+
+ifndef::openshift-rosa[]
+If you need to update the `endpointPublishingStrategy` value after your clister is deployed, you can configure the following `endpointPublishingStrategy` fields:
+
 * `hostNetwork.protocol`
 * `nodePort.protocol`
```

Comment:
better to add  `private.protocol` for the Private endpointPublishingStrategy

Reply:
Thanks, @ShudiLi . Updated.